component/attr_cache/attr_cache.go

```diff
@@ -525,12 +491,10 @@ func (ac *AttrCache) cacheAttributes(pathList []*internal.ObjAttr) {
 		ac.cacheLock.Lock()
 		defer ac.cacheLock.Unlock()
 		for _, attr := range pathList {
-			// TODO: will this cause a bug when cacheDirs is enabled?
-			if len(ac.cacheMap) > ac.maxFiles {
```

Comment:
Are we going to remove the cache entry limit entirely as a feature?
If so, we should probably remove the associated variables (`defaultMaxFiles`, `ac.maxFiles`), and maybe even the config option (`max-files`).
@jfantinhardesty, do you have an opinion on this?